Tests/MeiliSearchTests/MockURLSession.swift

```diff
@@ -57,3 +72,13 @@ class MockURLSessionDataTask: URLSessionDataTaskProtocol {
         resumeWasCalled = true
     }
 }
+
+private let ERROR = 0
+private let SUCCESS = 1
```

Comment:
Is it not the contrary?
In general, we return 0 when succeeded, and 1 when failed.

Reply:
You are correct, I was thinking about `Bool(1) == true` and `Bool(0) == false`.

Reply:
Haha, I understand the confusion 😉

Reply:
So, can you change that @ppamorim? 
```
ERROR = 1
SUCCESS = 0
```
or
```
TRUE = 1
FALSE = 0
```

Reply:
@curquiza I ditch this and implemented a profer solution using the enum `ResponseStatus` with `success and `error`.

Reply:
I like your solution!